plugins/GSdx/GSRenderer.cpp

```diff
@@ -274,7 +287,7 @@ bool GSRenderer::Merge(int field)
 
 		if(m_regs->SMODE2.INT && m_interlace > 0)
 		{
-			if (m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
+			if(m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
```

Comment:
Because same as you `can't help but think that the variable might be of non-bool type.`
You can argue that one  `can't help but think that the variable might be of bool type.` if it's `if (pointer)` instead of `if (pointer != nullptr)`.